packages/realm/src/app-services/Sync.ts

```diff
@@ -172,33 +171,21 @@ export class Sync {
   }
 
   /**
-   * The default OpenRealmBehaviorConfiguration used for local realms.
-   * This is used if the behavior is not specified by the developer when opening the realm.
-   * @internal
+   * The default behavior settings if you want to open a synchronized Realm immediately and start working on it.
+   * If this is the first time you open the Realm, it will be empty while the server data is being downloaded in the background.
+   * @deprecated since v12
    */
-  static defaultLocalOpenRealmConfiguration: Readonly<OpenRealmBehaviorConfiguration> = {
+  static openLocalRealmBehavior = {
     type: OpenRealmBehaviorType.OpenImmediately,
   };
```

Comment:
Removing `Readonly<OpenRealmBehaviorConfiguration>` might have been accidental? (Same for `static downloadBeforeOpenBehavior`)

Also, see earlier comment regarding having this reference what is being used internally as well.

Reply:
Yes, that was accidental.